grammar.js

```diff
@@ -926,14 +918,15 @@ module.exports = grammar({
       '}'
     ),
 
+    tuple_item_pattern: $ => seq(
+      repeat($.decorator),
+      $._pattern,
+      optional($.type_annotation),
+    ),
```

Comment:
This new rule is to ensure that `$.type_annotation` not to be a `tuple_item_pattern` node in `tuple_pattern`